lib/config/presets/index.ts

```diff
@@ -127,6 +128,18 @@ export function parsePreset(input: string): ParsedPreset {
     } else {
       presetName = str.slice(1);
     }
+  } else if (str.includes('//')) {
+    // non-scoped namespace with a subdirectory preset
+    const re = /^([\w/.]+?)\/\/(?:([\w./]+)\/)?([\w.]+)$/;
```

Comment:
```suggestion
    const re = /^([\w./]+?)\/\/(?:([\w./]+)\/)?([\w.]+)$/;
```

Just a small chnage to make both groups identical